Description: Qt only build.
Author: Fathi Boudra <fabo@debian.org>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4,7 +4,14 @@ enable_testing()
 
 find_package(KDE4)
 
+# Qt only build
 if(NOT KDE4_FOUND)
+    cmake_minimum_required(VERSION 2.6.2 FATAL_ERROR)
+    # set the cmake policies to the 2.4.x compatibility settings
+    cmake_policy(VERSION 2.4.5)
+    # CMP0002: multiple targets with the same name for the unit tests
+    cmake_policy(SET CMP0002 OLD)
+    find_package(Qt4)
     set(DATA_INSTALL_DIR share)
 endif(NOT KDE4_FOUND)
 
@@ -20,4 +27,6 @@ add_subdirectory(include)
 add_subdirectory(kdev-pg)
 add_subdirectory(examples EXCLUDE_FROM_ALL)
 
-add_subdirectory(tests)
+if(KDE4_FOUND)
+    add_subdirectory(tests)
+endif(KDE4_FOUND)
